packages/jest-environment-enzyme/package.json

```
@@ -43,12 +43,10 @@
   "peerDependencies": {
     "enzyme": "3.x",
     "jest": ">=22.0.0",
-    "react": "^0.13.0 || ^0.14.0 || ^15.0.0 || >=16.x"
+    "react": "^0.13.0 || ^0.14.0 || ^15.0.0 || >=16.x",
+    "jest-environment-jsdom": ">=22.0.0"
```

This shouldn’t use >=, because you can’t guarantee compact with future majors.
```suggestion
    "jest-environment-jsdom": "^ 25.2.0"
```

---

I was following the pattern already set by the `jest` peer dependency with using the `>=`. Should we also change the `jest` peer dependencies?

---

Oof, true enough.

Ideally yes, jest should be using `^22 || ^23` etc, and that should probably match this line.

---

However, changing the jest peer dep like that is a breaking change at this point - but so is adding a new peer dep.

---

In an idealistic world I completely agree with you. However on a more practical level, it's situations like https://github.com/reduxjs/redux-devtools/issues/473 (one example of many in my experience) that cause me to want to be more permissive.

@blainekasten What do you think? What do you want the the peer dependency ranges to be?

---

@ljharb I made a PR to tighten peer dependency ranges: https://github.com/FormidableLabs/enzyme-matchers/pull/344. If that PR gets merged before this one, then I'll gladly update the dependency range here. Otherwise, if this PR gets merged first, then I'll fix it in the other PR.

Since the `>=` is consistent with the status quo, I think it should be fine for this PR to be merged as-is. Conversation about the version ranges can happen on the other PR.

---

I don't think consistency is as important here as getting the range of a new peer dep correct; while it's not up to me, i wouldn't merge it with `>=`.